Sources/SwiftInspectorVisitors/TypeDescription.swift

```diff
@@ -267,9 +303,8 @@ public enum TypeDescription: Codable, Hashable {
 extension TypeSyntax {
 
   /// Returns the type description for the receiver.
-  /// - Warning: Do not call on a type syntax node of type `ClassRestrictionTypeSyntax`,
```

Comment:
this warning got out-of-date! Turns out including the same information twice can lead to inconsistencies 😬